src/dom-to-image-more.d.ts

```
@@ -1,10 +1,10 @@
-    import domToImage = require('dom-to-image');
-    
-    namespace domToImage {
-        function toPng(node: Node, options: any): Promise<string>;
-    }
+declare module "dom-to-image-more" {
+	import domToImage = require("dom-to-image");
 
-    export = domToImage;
-}
\ No newline at end of file
+	namespace domToImage {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'domToImage'. Consider to delete it or rename it.

'domToImage' is defined here:


(lint/suspicious/noRedeclare)